src/Sylius/Bundle/CoreBundle/Fixture/Factory/AdminUserExampleFactory.php

```diff
@@ -44,11 +47,13 @@ class AdminUserExampleFactory extends AbstractExampleFactory implements ExampleF
 
     public function __construct(
         FactoryInterface $userFactory,
+        FactoryInterface $avatarImageFactory,
```

Comment:
This service should be injected the same way as `FileLocatorInterface` & `ImageUploaderInterface`. Also placed on the last line, to support BC.

Reply:
Ah ok I'll do that ASAP